bigquery_etl/format_sql/tokenizer.py

```diff
@@ -433,6 +487,11 @@ class FieldAccessOperator(Operator):
     AngleBracketKeyword,
     SpaceBeforeBracketKeyword,
     ReservedKeyword,
+    JinjaStatementLine,
+    JinjaStatementMiddle,
+    JinjaStatementStart,
+    JinjaStatementEnd,
+    JinjaExpression,
```

Comment:
these are getting complicated. I think we should move these to separate token priority list `BIGQUERY_JINJA_TOKEN_PRIORITY`, and require an option to enable jinja support when formatting.